bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -232,42 +242,50 @@ export default Whiteboard = React.memo(function Whiteboard(props) {
           typeName: remoteShape.typeName,
         };
 
-        // Compare each property
-        Object.keys(remoteShape).forEach((key) => {
-          if (
-            key !== "isModerator" &&
-            !isEqual(remoteShape[key], localShape[key])
-          ) {
-            diff[key] = remoteShape[key];
-          }
-        });
-
-        if (remoteShape.props) {
-          Object.keys(remoteShape.props).forEach((key) => {
-            if (!isEqual(remoteShape.props[key], localShape.props[key])) {
-              diff.props = diff.props || {};
-              diff.props[key] = remoteShape.props[key];
+        if (!selectedShapeIds.includes(diff?.id) && prevShapesRef.current[`${diff?.id}`].meta?.updatedBy !== currentUser?.userId) {

```

Comment:
Would you consider storing `diff?.id` to a variable? This way you could add a check for it in the beginning of the `if` condition check. Also, seems we may be missing the case of `prevShapesRef.current[`${diff?.id}`]` being undefined - i.e. if there is no current with that index